frontend/packages/console-shared/src/components/terminal/Terminal.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import TerminalBody from './TerminalBody';
+import TerminalDrawer from './TerminalDrawer';
+import { Button } from '@patternfly/react-core';
+
+const Terminal: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  return (
+    <>
+      <Button variant="control" onClick={() => setOpen(!open)}>
+        Open Drawer
+      </Button>
+      {open && (
```

Comment:
You are losing your height state when you unmount this high up in the component tree.

Reply:
This is temporary and will be changed once we implement the actual terminal.